lib-utilities/common/mockconfig.go

```diff
@@ -65,6 +65,12 @@ func SetUpMockConfig() error {
 	}
 	config.Data.AddComputeSkipResources = &config.AddComputeSkipResources{
 		SystemCollection: []string{
+			"Chassis",
+			"Managers",
+			"LogServices",
+		},
+		ManagerCollection: []string{
+			"Systems",
 			"Chassis",
 			"LogServices",
```

Comment:
Are we sure LogServices is a manager?  What does Systems/ManagerCollection do ?

Reply:
These collections are used to remove the links from the list of links present for that particular collection.
for e.g. - Managers links will be removed from list of links retrieved from System URI's and manager will be handled independently

Reply:
give me meaningful names.

Reply:
Addressed comments - Modified the list names